### PR TITLE
expand partial_eval to take boolean

### PR DESCRIPTION
**Pull request by yamaguchi1024**

### CODE REVIEW

File: SYS_ATL/LoopIR_scheduling.py [hunk 1]
```diff
@@ -553,12 +553,12 @@ def __init__(self, proc, arg_vals):
 
         # Validate env:
         for k, v in self.env.items():
-            if not arg_types[k].is_indexable():
+            if not arg_types[k].is_indexable() and not arg_types[k].is_bool():
                 raise SchedulingError("cannot partially evaluate "
-                                      "numeric (non-index) arguments")
-            if not isinstance(v, int):
+                                      "numeric (non-index, non-bool) arguments")
+            if not isinstance(v, int) and not isinstance(v, bool):
```
Comment (alexreinking) [inline SYS_ATL/LoopIR_scheduling.py:559]: Bool is a subtype of int in Python. This should just be `not isinstance(v, int)` and a comment should be left about intentionally excluding bool, too.

**Review by alexreinking — CHANGES_REQUESTED** [review]
One bit of code cleanup needed, but otherwise LGTM

File: SYS_ATL/LoopIR_scheduling.py (hunk 1 above)
Comment (yamaguchi1024) [reply SYS_ATL/LoopIR_scheduling.py:559]: I see, that makes sense

**Review by alexreinking — APPROVED** [review]

**Comment by alexreinking** [comment]
LGTM - go ahead and merge when the checks pass

**Comment by yamaguchi1024** [comment]
> LGTM - go ahead and merge when the checks pass

Thanks!

**Pull request merged**